lib/active_fedora/file.rb

```diff
@@ -139,6 +140,19 @@ def metadata?
       false
     end
 
+    def external_content?
+      external_url.present?
+    end
+
+    def external_url
+      return nil unless mime_type.start_with? "message/external-body"
+      mime_type[/url=['"](.+?)['"]/i, 1]
+    end
+
+    def external_url=(external_file_url)
+      self.mime_type = "message/external-body; access-type=URL; URL=\"#{external_file_url}\""
```

Comment:
@awead Right — the current Fedora impl will only send a redirect for `access-type="URL"` and a HTTP URL.  With any other access-type, it'll just serve the internal binary (0-byte or otherwise).

But this behavior is under active discussion in the Fedora API spec group, so I wouldn't want to bake anything else in right now.  So my proposal is:

* If `content-type` starts with `message/external-body`, the `access-type` parameter is `URL` and the `URL` parameter contains something that looks like a URL, then treat it as external content.
* Everything else is treated as a normal binary.

When using `file:///` URLs and/or `access-type="local-file"` get specified, then we can add support for those as needed.